configure.ts

```diff
@@ -25,8 +25,8 @@ export async function configure(command: ConfigureCommand) {
   /**
    * Publish config file
    */
-  await codemods.makeUsingStub(stubsRoot, 'stubs/config/kafka.stub', {})
-  await codemods.makeUsingStub(stubsRoot, 'stubs/start/kafka.stub', {})
+  await codemods.makeUsingStub(stubsRoot, 'config/kafka.stub', {})
+  await codemods.makeUsingStub(stubsRoot, 'start/kafka.stub', {})
```

Comment:
I have no idea why I needed to change these, but after adjusting tsc config, suddenly it didn't find the files correctly. Looking at other packages, and it seems I shouldn't have included the `stubs/` originally.

Reply:
yeah, I had the reverse when I messed with this last, it wouldn’t work _without_ `stubs/` even though none of the examples I based this off had them and I tried this in an act of desperation. I’m glad this work correctly now.